src/components/RadioBorderedBox/index.js

```diff
@@ -3,12 +3,13 @@ import { transparentize } from 'polished'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Badge from '../Badge'
-import BorderedBox from '../BorderedBox'
 import Box from '../Box'
 import Radio from '../Radio'
 import Typography from '../Typography'
 
-const StyledBorderedBox = styled(BorderedBox)`
+const StyledBox = styled(Box, {
+  shouldForwardProp: prop => !['bordered'].includes(prop),
```

Comment:
You need to pass the bordered props to Box no ?

Reply:
Yes of course sorry :)